### PR TITLE
adding a full JSON file

### PR DESCRIPTION
**Pull request by choldgraf**

### CODE REVIEW

**Comment by choldgraf** [comment]
@yuvipanda @aculich WDYT about this?

File: doc/setup-binderhub.rst [hunk 1]
```diff
@@ -44,7 +44,20 @@ Create a file called ``secret.yaml`` and enter the following::
         secretToken: "<output of SECOND `openssl rand -hex 32` command>"
   registry:
     password: |
-      <contents of the JSON file for the container registry from Service Accounts>
+      <content of the JSON file downloaded earlier for the container registry from Service Accounts>
+      <it will look something like:>
+      {
+      "type": "service_account",
+      "project_id": "binderhub-test-2017-10-18",
```
Comment (yuvipanda) [inline doc/setup-binderhub.rst]: remove value here.

File: doc/setup-binderhub.rst [hunk 2]
```diff
@@ -44,7 +44,20 @@ Create a file called ``secret.yaml`` and enter the following::
...
+      "private_key_id": "6733ae2c9ff2a6b159b068729efdce0b6a692329",
```
Comment (yuvipanda) [inline doc/setup-binderhub.rst]: remove the value here?

**Comment by choldgraf** [comment]
@willingc what do you think about this? Getting people to copy/paste the JSON properly has always been a pain point, so this is trying to make that a little bit clearer...

**Comment by willingc** [comment]
LGTM @choldgraf

**Pull request merged**